mam/api/BUILD

```diff
@@ -31,3 +39,18 @@ map_generate(
     parent_directory = "mam/api",
     value_type = "mam_msg_read_context_t",
 )
+
+# Build command:
+# bazel clean (recommended)
+# bazel build --spawn_strategy=local --define=workspace=$(bazel info workspace)  --define=output_base=$(bazel info output_base)  --config=emscripten //mam:mam.js
+
+emcc_binary(
+    name = "api.js",
+    srcs = [
+        "api.c",
+    ],
+    defines = ["MAM_EXPORT_JS"],
```

Comment:
remove

Reply:
@tsvisabo Maybe that solve the issue of needing to have it under /mam/api ?

Reply:
but how then where would you define `__EMSCRIPTEN__`?

Reply:
I believe @oopsmonk suggested it's defined by emscripten compilers, right ?

Reply:
could be, i'll check as soon as my pc is back to normal

Reply:
it's predefined in the toolchain.

Reply:
normally, compilers will define some flags or macros for platforms and itself that can help to distinguish the environment. we should use it, not define a new one by us.